docs/getting-started.rst

```diff
@@ -79,6 +87,14 @@ displayed using this command::
                           Set amount of displayed information
     -s, --stop-on-failure
                           Stop running checks after a failed one.
+    --map-charm MAP_CHARM
+                          WARNING: This option can lead to failed verifications
+                          when used incorrectly. This option allows user to
+                          explicitly specify which charm is the application
+                          running. It is useful when charm is deployed from
+                          local source or from non-official charmhub repository.
+                          Expected value format is <APP_NAME>:<CHARM_NAME>. For
```

Comment:
is "CHARM_NAME" the same as "SUPPORTED_CHARM" or "OFFICIAL_CHARM_NAME"? If an application has been deployed using a local source or for e.g. `openstack-charmers-ceph-osd`, the mapping should match the supported verifier, right?

I think we should make it more explicit for avoidance of doubt. "<APP_NAME>:<CUSTOM_CHARM_NAME>" would never work because it is not listed in the list of supported charms.

PS: Confirmed by `verifier = SUPPORTED_CHARMS.get(charm)` line in the `verifier/__init__.py` script.

Reply:
Yes, the `CHARM_NAME` in this case has to be name of the charm that we support. I tried to allude to this fact by the following sentence that tells user how to get list of supported charms. If you think that it's not entirely clear, we can change it to `<APP_NAME>:<SUPPORTED_CHARM_NAME>` but I tried to keep it as short as possible as this is a `--help` output and it's already bit too long for my liking :smile:

Reply:
I'm all in for shorter instructions. Maybe the description section should mention it (that is must be one of the supported charms). That would be enough.